py4DSTEM/process/virtualimage/virtualimage.py

```diff
@@ -85,7 +85,36 @@ def get_virtualimage_ann(datacube, x0, y0, Ri, Ro, verbose=True):
     return virtual_image
 
 
+def get_virtualimage(
+    datacube, 
+    mask, 
+    verbose=True,
+    return_mask_coords=False,
```

Comment:
This is unused?

Reply:
Oops a leftover from the previous version!